### ISSUE TITLE
Write Blob, instead of just string 

### ISSUE THREAD

**Issue by ShivamJoker** [alpha0010/react-native-file-access#78]
**Feature**
I am creating an `XLSX` file and I would like to save it in the file system.

But right now the only way is to convert it to string, which might not work since it's a compressed files.

**Motivation**
It can be used to save images and other media files too.

**Comment by ShivamJoker** [comment]
I tried the base64 method but it says `bad base64`

This is what my string looks like:
```
data:application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;base64,UEsDBBQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

**Comment by alpha0010** [comment]
Good idea.  Not sure when I will have time to implement though, as I am quite busy currently (pull requests always welcome).

Until then, you can use a workaround to get it working in your app.  Trouble is that string is a [data URL](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs), not pure base64 data. Here are a [variety of ways](https://stackoverflow.com/questions/18650168/convert-blob-to-base64) to convert the blob so it will work with base64 methods of this library.
